etc/profile-a-l/display.profile

```diff
@@ -40,7 +40,8 @@ shell none
 private-bin display,python*
 private-dev
 # On Debian-based systems, display is a symlink in /etc/alternatives
-private-etc alternatives,ld.so.cache,ld.so.preload
+private-etc alternatives,ImageMagick*,ld.so.cache,ld.so.preload
```

Comment:
private-etc doesn't have globbing support (yet):
```
$ firejail /usr/bin/display
Error: "ImageMagick*" is an invalid filename: rejected character: "*"
Error: proc 612189 cannot sync with peer: unexpected EOF
Peer 612190 unexpectedly exited with status 1
```
Using `ImageMagick-7` in the private-etc line works fine here on Arch Linux.

Reply:
> private-etc doesn't have globbing support (yet):

Ah, right you are. I had it correct in my `display.local` for testing and then did it wrong when preparing the PR :/

Updated.